cli/tests/integrations/test_task.py

```diff
@@ -25,12 +25,14 @@
              (SLEEP1, 'test-app1'),
              (SLEEP2, 'test-app2'))
 NUM_TASKS = len(INIT_APPS)
+TASK_ID_COMPLETED = ''
 
 
 def setup_module():
     # create a completed task
     with app(SLEEP_COMPLETED, 'test-app-completed'):
-        pass
+        global TASK_ID_COMPLETED
+        TASK_ID_COMPLETED = _get_completed_task_id()
```

Comment:
How would I get the `TASK_ID_COMPLETED` outside of `setup_module` ? Only point in the code where its is running is in `setup_module` right ? By TASK_ID_COMPLETED,  I mean to get the full ID of the task including hash. 
I did this for `test_ls_completed` not to fail. It used to succeed in p27 tests and fail in p34 tests because at that point there would be multiple `test-app-completed` and I wanted a way to get just one of those.